native-assets/NFT/README.md

```diff
@@ -0,0 +1,116 @@
+
+## Standards Implementations
+
+The project implements and follows the [SRC-20; Token](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20), [SRC-3; Mint and Burn](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_3), and [SRC-7; Metadata](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_7) standards. It also uses the [Token Library](https://github.com/FuelLabs/sway-libs/tree/master/libs/token) to implement the basic functionality behind the standards.  
+
+### SRC-20
+
+The [SRC-20](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20) standard requires that there is a maximum number of one token per NFT asset. It also states that the decimals must be `0u8` for any NFT. This project conforms to both these restrictions and thus can be deemed an NFT on the Fuel Network. 
+
+Set functions for name and symbol have been provided to the user. While traditionally name and symbol are written into the contract rather than storage, this contract is open to mint new types of assets. This means that every NFT minted by this contract may contain a different name and symbol. 
+
+The [SRC-20](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20) ABI defined below has also been implemented.
+
+```sway
+abi SRC20 {
+    #[storage(read)]
+    fn total_assets() -> u64;
+    #[storage(read)]
+    fn total_supply(asset: AssetId) -> Option<u64>;
+    #[storage(read)]
+    fn name(asset: AssetId) -> Option<String>;
+    #[storage(read)]
+    fn symbol(asset: AssetId) -> Option<String>;
+    #[storage(read)]
+    fn decimals(asset: AssetId) -> Option<u8>;
+}
+```
+
+### SRC-3
+
+The [SRC-3](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_3) standard defines the ABI for minting and burning. This has been properly implemented.
+
+```sway
+abi SRC3 {
+    #[storage(read, write)]
+    fn mint(recipient: Identity, sub_id: SubId, amount: u64);
+    #[storage(read, write)]
+    fn burn(sub_id: SubId, amount: u64);
+}
+```
+
+### SRC-7
+
+The [SRC-7](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_7) standard defines the ABI for retrieving metadata. This has been properly implemented. 
+
+A set function that uses storage has been provided to allow the user to set their own desired metadata. There is no limit or restrictions to what and the amount of metadata an asset may have.
+
+```sway
+abi SRC7 {
+    #[storage(read)]
+    fn metadata(asset: AssetId, key: String) -> Option<Metadata>;
+}
+```
+
+## Project structure
+
+The project consists of a smart contract.
+
+<!--Only show most important files e.g. script to run, build etc.-->
+
+```sh
+NFT
+├── project
+│   ├── contracts
+│   │   └── NFT-contract
+│   │       └──src/main.sw
+│   └── SPECIFICATION.md
+├── ui
+│   └── SPECIFICATION.md
+└── README.md
```

Comment:
This looks to be out of date or rather the project structure in this PR does not conform to this